internal/spentenergy/spentenergy.go

```diff
@@ -14,16 +15,47 @@ const (
 
 func WalkingSpentCalories(steps int, weight, height float64, duration time.Duration) (float64, error) {
 	// TODO: реализовать функцию
+	if steps <= 0 {
+		return 0, errors.New("numbers of the steps<=0")
+	}
+	if weight <= 0 {
+		return 0, errors.New("weight<=0")
```

Comment:
лучше писать суть ошибки
```suggestion
		return 0, fmt.Errorf("weight must be greater than zero")
```